lib/fetch-docs.js

```diff
@@ -46,6 +46,8 @@ var fetchDocs = module.exports = function fetchDocs (_settings, callback) {
 
     // override tarball for testing out branches:
     // tarballUrl = 'https://api.github.com/repos/electron/electron/tarball/master'
+    // tarballUrl = 'https://github.com/electron/electron/archive/ba49fbc8ba2fac0b72b1530f2ce46c1cd3110620.tar.gz'
+    // version = 'v1.4.1'
```

Comment:
Good point. Cleaned that up a bit. I just left it there because I always have a hard time finding/remembering the URL structures for fetching repo tarballs.